core/admin/editor-one-menu/elementor-one-menu-manager.php

```diff
@@ -292,10 +291,6 @@ public function intercept_legacy_submenus(): void {
 	}
 
 	public function enqueue_admin_menu_assets(): void {
-		if ( ! $this->menu_data_provider->is_elementor_editor_page() ) {
```

Comment:
@ManorHazaz this needs to be removed as it was preventing the flyout from displaying.